pkg/compare/testdata/DiffinCustomOmittedFieldsIsntShownLeadingDot/reference/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+Parts:
+  - name: ExamplePart
+    Components:
+      - name: Dashboard
+        type: Required
+        requiredTemplates:
+          - path: deploymentMetrics.yaml
+
+fieldsToOmit:
+  defaultOmitRef: deployment
+  items:
+    deployment:
+      - pathToKey: .spec.selector.matchLabels.k8s-app
```

Comment:
curios, why did you decide to support a leading dot?

Reply:
some of the similar syntaxes have a leading dot. And its really easy to just ignore it. Didn't see the point in not :)